toolkits/global/packages/global-forms/scss/00-tokens/_default.tokens.scss

```diff
@@ -0,0 +1,34 @@
+// Generated on 30/06/2022, 16:20:58
+// Source: design-tokens/componenets/global/global-forms/default.json
```

Comment:
Spelling mistake: `componenets` should be `components`
across this and springer

Reply:
whoops! 

I had a bit of a re-think on this, as an aside. 

The current `npm scripts` for generating the Sass variables are a pretty blunt tool in generating `literal` and `alias` tokens (pretty much) every time. 

I'm going to rework it so that they're either _very_ separate tasks or the script checks for changes to disc for the`.json` files and only compiles things that have changed. 

Moving forward, it should be the case that `literal tokens` very rarely get changed. `alias tokens` do get changed but not often, and `component` tokens` only get changed/added to when there are changes from the `alias` tokens or we are introducing `component` tokens to the component.

If that makes sense.